### PR TITLE
Spawning indicator

### PR DESCRIPTION
**Pull request by Simonolsson98**
- Should add a spawning indicator for each formation that will be resized when you change amountOfTroops
- It's only approximate for now

### CODE REVIEW

**Comment by Simonolsson98** [comment]
Noticed a slight bug with spawning positions, fixing right now!

File: KA-Flocking/Assets/Scripts/EntitySpawning.cs [hunk 1]
```diff
@@ -34,35 +40,46 @@ public class EntitySpawning : MonoBehaviour
     private Material previousMaterial;
     private FlockAgent previousCastle = null;
     public Material castleRangePreviewMaterial;
-
-    void Start(){
+    void Start() {
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/EntitySpawning.cs:43]: Boi why do this. Spacing is king

File: KA-Flocking/Assets/Scripts/EntitySpawning.cs [hunk 2]
```diff
@@ -34,35 +40,46 @@ public class EntitySpawning : MonoBehaviour
...
+            formationAreaArray[1] = rectAreaToSpawn;
+            formationAreaArray[2] = triAreaToSpawn;
+            initTriangleIndicator();
+            setIndicatorColors(formationAreaArray);
+            if (SceneManager.GetSceneByName("PlayerOneSetupScene").isLoaded) {
+                flock = GameObject.Find("Team 1 Flock").GetComponent<Flock>();
+            } else if (SceneManager.GetSceneByName("PlayerTwoSetupScene").isLoaded) {
+                flock = GameObject.Find("Team 2 Flock").GetComponent<Flock>();
+            }
+            money.text = "Money: " + flock.moneyAmount.ToString();
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/EntitySpawning.cs:62]: boi asså. Dina metoder behöver nedbrytning, uppdelning med newlines och kommentarer. Big smh

File: KA-Flocking/Assets/Scripts/EntitySpawning.cs [hunk 3]
```diff
@@ -81,30 +98,97 @@ public class EntitySpawning : MonoBehaviour
...
+    private void initTriangleIndicator(){
+        triAreaToSpawn.AddComponent<MeshFilter>();
+        triAreaToSpawn.AddComponent<MeshRenderer>();
+        triangleMesh = triAreaToSpawn.GetComponent<MeshFilter>().mesh;
+        triangleMesh.Clear();
+        triangleMesh.vertices = new Vector3[] { new Vector3(0, 0, 0), new Vector3(0, 0, 0), new Vector3(0, 0, 0) };
+        triangleMesh.uv = new Vector2[] { new Vector2(0, 0), new Vector2(0, 1), new Vector2(1, 1) };
+        triangleMesh.triangles = new int[] { 0, 1, 2 };
+    }
+    private void setIndicatorColors(GameObject [] formationAreaArray) {
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/EntitySpawning.cs:136]: I don't really care but have some space between the methods

**Pull request merged**